numpy/core/src/multiarray/dragon4.c

```diff
@@ -654,7 +666,7 @@ BigInt_Pow10(BigInt *result, npy_uint32 exponent)
      * initialize the result by looking up a 32-bit power of 10 corresponding to
      * the first 3 bits
      */
-    smallExponent = exponent & 0x7;
+    smallExponent = exponent & bitmask_u32(3);
```

Comment:
seems more readable to spell this as 0x7

Reply:
Writing it this way makes the code match the comment